static/css/styles.css

```diff
@@ -20,49 +16,26 @@
 
 :root {
   --radius: 8px;
-  --bg-color: #f0efea;
```

Comment:
Dark-specific and light-specific properties have been moved to `.dark-mode-on:root` and `.dark-mode-off:root` as well as their respective media query equivalents.